libraries/ios/WooCommerceShared/WCReactNativeViewController.h

```diff
@@ -14,14 +14,8 @@ NS_ASSUME_NONNULL_BEGIN
                              appPassword: (NSString*) appPassword;
 
 -(instancetype)initWithBundle:(NSURL *) url
-            analyticsProvider:(id<WCRNAnalyticsProvider>) analyticsProvider
-                       blogID:(NSString *)blogId
-                     apiToken: (NSString*) apiToken;
+            analyticsProvider:(id<WCRNAnalyticsProvider>) analyticsProvider;
 
--(instancetype)initWithBundle:(NSURL *) url
-            analyticsProvider:(id<WCRNAnalyticsProvider>) analyticsProvider
-                      siteUrl:(NSString *)siteUrl
-                  appPassword: (NSString*) appPassword;
```

Comment:
❓ won't this change break embedding React Native to WCiOS app? How WCiOS app will send authentication credentials to React Native now?

Reply:
oh okay, thank you for explanation 👍